README.md

```diff
@@ -78,11 +82,18 @@ To compile, starting at the project root directory, create the build directory a
 ```
 mkdir build && cd build && cmake ..
 ```
-Optionally, you can specify the install location by providing the desired install path prefix:
+On MacOS, you may also need to provide the path to openssl (your exact path may vary):
 ```
-cmake .. -DCMAKE_INSTALL_PREFIX=/install/path
+cmake -DOPENSSL_ROOT_DIR=/usr/local/Cellar/openssl/1.0.2s -DOPENSSL_LIBRARIES=/usr/local/Cellar/openssl/1.0.2s/lib ..
 ```
 
+Other build flags include:
+| Flag | Value | Description |
```

Comment:
Thanks for starting on flags! Can we actually do these README updates in a separate PR? (I think this way of updating the SSL root directory is more likely to break than whats on staging atm)

Reply:
I'm not sure what you mean. Do you want me to separate the README changes to a different PR? Or do you want to add more README changes than the ones in this PR?

Reply:
Yeah I mean lets separate the README changes into a separate PR

Reply:
I can still keep the .gitignore changes right?